frontend/test/components/subview.spec.jsx

```diff
@@ -1,8 +1,11 @@
 import { configure, mount } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
 import { BrowserRouter as Router } from 'react-router-dom';
+import { URLSearchParams } from 'url';
 import { SubViewComponent } from '../../ui/subview';
 
+global.URLSearchParams = URLSearchParams;
```

Comment:
Is this necessary?

Reply:
Yes, it is required, as URLSearchParams is not defined for our current version of jest.

Reply:
Interesting, right, I guess it's a browser global.